SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialGDKSpanId.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/Connection/SpatialGDKSpanId.h"
+
+// ----- FSpatialGDKSpanId -----
+
+FSpatialGDKSpanId::FSpatialGDKSpanId()
+	: bIsValid(false)
```

Comment:
I think we can remove this and use `Trace_SpanId_Equal(Trace_SpanId_Null(), Data)` to signify the valid state - this allows this class to be sized as TRACE_SPAN_ID_SIZE_BYTES and would make a lot of the management here much simpler. We can then pass this class directly to the worker API without introducing memory layout changes

Reply:
Trace_SpanId_Equal(Trace_SpanId_Null(), Data) is a valid spanId. Non null ids are also valid.

However I think we might be able to get rid of the IsValid check altogether and just use null spanid. Will think about this.